x/multi-staking/keeper/proposal.go

```diff
@@ -37,6 +40,41 @@ func (k Keeper) AddMultiStakingCoinProposal(
 	return nil
 }
 
+// AddMultiStakingEVMCoinProposal handles the proposals to add a new bond token
+func (k Keeper) AddMultiStakingEVMCoinProposal(
```

Comment:
What about a proposal to remove the coin?

Reply:
we have UpdateBondWeight to do that